src/librustdoc/clean/types.rs

```diff
@@ -82,13 +82,19 @@ crate struct ExternalCrate {
 /// directly to the AST's concept of an item; it's a strict superset.
 #[derive(Clone)]
 crate struct Item {
-    /// Stringified span
-    crate source: Span,
-    /// Not everything has a name. E.g., impls
+    /// The [`Span`] of this item in the source code.
+    crate span: Span,
+    /// The name of this item.
+    /// Optional because not every item has a name, e.g. impls.
     crate name: Option<Symbol>,
+    /// Attributes on this item, e.g. `#[derive(...)]` or `#[inline]`.
     crate attrs: Box<Attributes>,
```

Comment:
Does this add any info you can't find from the `Attributes` link?

Reply:
Likewise, `Attributes` has no docs.